snippets/snippets.json

```diff
@@ -155,5 +155,58 @@
         "const MOCK_ROUTER_PROVIDER = { provide: Router, useClass: MockRouter };",
         ""
       ]
+    },
+    "Using HttpTestingController for success request": {
+      "prefix": "jat-success-http-testing-controller-spec-expecting-one-request",
```

Comment:
Shorter prefix and longer description is preferable.

jat-http-with-success
jat-http-with-error
jat-dispatch-event


Ex: 
"description": "Spec which makes use of angular HttpTestingController to test http requests. We can return a mock data when the request matches a Url",

Reply:
Yes updated prefix and description.